cpp/benchmarks/copying/contiguous_split_benchmark.cu

```diff
@@ -34,10 +34,12 @@ void BM_contiguous_split_common(benchmark::State& state,
                                 int64_t bytes_total)
 {
   // generate splits
-  cudf::size_type split_stride = num_rows / num_splits;
   std::vector<cudf::size_type> splits;
-  for (int idx = 0; idx < num_rows; idx += split_stride) {
-    splits.push_back(std::min(idx + split_stride, static_cast<cudf::size_type>(num_rows)));
+  if (num_splits > 0) {
+    cudf::size_type split_stride = num_rows / num_splits;
+    for (int idx = 0; idx < num_rows; idx += split_stride) {
+      splits.push_back(std::min(idx + split_stride, static_cast<cudf::size_type>(num_rows)));
+    }
```

Comment:
Nit: this seems like a good place for a `splits.reserve` followed by a transform of a counting iterator.